flash/core/finetuning.py

```diff
@@ -0,0 +1,153 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import inspect
+from typing import List, Union
+
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks import BaseFinetuning
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from torch import nn
+from torch.optim import Optimizer
+
+_EXCLUDE_PARAMTERS = ["self", "args", "kwargs"]
```

Comment:
```suggestion
_EXCLUDE_PARAMTERS = ("self", "args", "kwargs")
```
so they are not mutable

Reply:
Done !